appveyor.yml

```diff
@@ -5,13 +5,13 @@ environment:
     - PlatformToolset: mingw-w64
       platform: mingw-w64
       APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2022
-      QTPath: C:\Qt\6.3.1\mingw_64
+      QTPath: C:\Qt\6.5\mingw_64
```

Comment:
maybe 6.6?

Reply:
See https://www.appveyor.com/docs/windows-images-software/#qt . Version 6.6 is not available yet. Why should be 6.6 used already now as the official used version seems to be 6.5.3 according to

https://github.com/Governikus/AusweisApp/blob/community/libs/Versions.cmake#L1

Reply:
Ok, it doesn't matter. Just an idea...